In [ ]:
import numpy as np
import sympy as sy

a = np.array([[1, -1],[6, -4]])
a

In [ ]:
np.linalg.eig(a)

(array([-1., -2.]), array([[0.4472136 , 0.31622777],
        [0.89442719, 0.9486833 ]]))

didapat bahwa eigenvalue -1 dengan eigenvactor 0.4472136, 0.89442719.sedangkan eigenvalue -2 dengan eigenvactor 0.31622777 dan 0.9486833.


